Bottom: b9eb01b0dbaaaa21cae0af6a2a275ae2270e974f
Top:    01deeaa91812355c88cf8d1a106d9197ad2b4884
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-23 23:43:19 +0000

ODDS AND SODS


---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index b0dcba0..44abe54 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -1510,7 +1510,7 @@ object3d_from_copper_layers_within_area (POLYAREA *area)
   /* DEBUG */
   poly_M_Copy0 (&PCB->Data->outline, info.poly);
   PCB->Data->outline_valid = true;
-  gui->invalidate_all ();
+//  gui->invalidate_all ();
 
   poly_Free (&info.poly);
 
diff --git a/src/polygon.h b/src/polygon.h
index aabc096..1343d03 100644
--- a/src/polygon.h
+++ b/src/polygon.h
@@ -34,7 +34,7 @@
 
 /* Implementation constants */
 
-#define POLY_CIRC_SEGS 10 //40
+#define POLY_CIRC_SEGS 8 //10 //40
 #define POLY_CIRC_SEGS_F ((float)POLY_CIRC_SEGS)
 
 #if 0
